grizli/tests/test_jwst.py

```diff
@@ -0,0 +1,31 @@
+"""
+Tests for JWST imaging and spectra, including 
+"""
+import os
+import glob
+import unittest
+
+import numpy as np
+from .. import utils, prep, multifit, fitting, GRIZLI_PATH
+
+
+class JWSTFittingTools(unittest.TestCase):
+    
```

Comment:
The test action is failing becauase it can't find the NIRISS configuration file(s).  This class should have a first method that checks for the necessary config files and fetch them if not, analogous to https://github.com/gbrammer/grizli/blob/master/grizli/tests/test_fits.py#L17.

This could be another option in [``utils.py``](https://github.com/gbrammer/grizli/blob/master/grizli/utils.py#L5729)  (`get jwst`) to add [jwst-grism-conf.tar.gz](https://github.com/gbrammer/grizli-config/blob/main/jwst-grism-conf.tar.gz) to the list of tarfiles to download and extract.

Reply:
I've tried to do this here: 8ef42e7f9e5469244df1ca98ce56384a634b1641 Let me know if it works